streams/catalog/src/main/java/com/hortonworks/streamline/streams/catalog/service/StreamCatalogService.java

```diff
@@ -2411,7 +2361,7 @@ public UDF addUDF(UDF udf) {
         return udf;
     }
 
-    public Map<String, Class<?>> loadUdfsFromJar(java.io.File jarFile) throws IOException {
+    public static Map<String, Class<?>> loadUdfsFromJar(java.io.File jarFile) throws IOException {
```

Comment:
Hm.  It fits very well with the other methods in StreamCatalogService, it would seem artificial to remove it to a utility class.  Perhaps it is merely fortunate that it is possible to change it to static, and it is the only method I needed to access from the mocked class.  But in the interests of making only minimal changes to enable unit testing, I think it is best to do it this way.